package.json

```diff
@@ -81,13 +81,15 @@
     "lint": "npm run lint:js && npm run lint:scss",
     "lint:js": "eslint --ext .js,.jsx .",
     "lint:scss": "stylelint src/**/*.scss",
+    "pack": "webpack -p",
     "precompile": "rm -rf lib",
     "prepare": "npm run compile",
-    "prepare-for-release": "terra prepare-for-release",
-    "start": "tt-serve",
-    "start-prod": "tt-serve -p",
-    "start-heroku": "tt-serve-static --port $PORT --site './build' ",
-    "start-static": "npm run compile:prod && tt-serve-static --site ./build",
+    "prepare-for-release": "npm run terra -- prepare-for-release",
```

Comment:
Does this command not work as `terra prepare-for-release`?

Reply:
updated here https://github.com/cerner/terra-application/pull/129/commits/ba236f374daa74bb3d075cd64ba16b3f6a9ac206